packages/protocol/contracts/governance/DowntimeSlasher.sol

```diff
@@ -35,88 +42,212 @@ contract DowntimeSlasher is SlasherUtil {
   }
 
   /**
-   * @notice Sets the slashable downtime
+   * @notice Sets the slashable downtime.
    * @param interval Slashable downtime in blocks.
    */
   function setSlashableDowntime(uint256 interval) public onlyOwner {
-    require(interval != 0, "Slashable downtime cannot be zero");
-    require(interval < getEpochSize(), "Slashable downtime must be smaller than epoch size");
+    require(interval != 0, "slashable downtime cannot be zero");
+    require(interval < getEpochSize(), "slashable downtime must be smaller than epoch size");
```

Comment:
Is there a reason to enforce this constraint anymore?

Reply:
Not after all the changes. Removing it!